.pluginsync.yml

```diff
@@ -0,0 +1,18 @@
+# File managed by pluginsync
+pluginsync_config: '0.1.8'
+managed_files:
+- .github
+- .github/ISSUE_TEMPLATE.md
+- .github/PULL_REQUEST_TEMPLATE.md
+- .gitignore
+- .pluginsync.yml
+- .travis.yml
+- CONTRIBUTING.md
+- LICENSE
+- Makefile
```

Comment:
This output is in conflict with the .sync.yml config (it should be ignored if it's unmanaged.)